src/Backend/Modules/ContentBlocks/Actions/ContentBlockAdd.php

```diff
@@ -43,27 +39,55 @@ public function execute(): void
         }
 
         /** @var CreateContentBlock $createContentBlock */
-        $createContentBlock = $form->getData();
-        $createContentBlock->userId = Authentication::getUser()->getUserId();
-
-        // The command bus will handle the saving of the content block in the database.
-        $this->get('command_bus')->handle($createContentBlock);
+        $createContentBlock = $this->createContentBlock($form);
 
         $this->get('event_dispatcher')->dispatch(
             ContentBlockCreated::EVENT_NAME,
-            new ContentBlockCreated($createContentBlock->contentBlock)
+            new ContentBlockCreated($createContentBlock->getContentBlockEntity())
         );
 
         $this->redirect(
-            BackendModel::createURLForAction(
-                'Index',
-                null,
-                null,
+            $this->getBackLink(
                 [
-                    'report' => 'added',
+                    'report' => 'content-block-added',
                     'var' => $createContentBlock->title,
                 ]
             )
         );
     }
+
+    private function createContentBlock(Form $form): CreateContentBlock
+    {
+        /** @var CreateContentBlock $createContentBlock */
+        $createContentBlock = $form->getData();
+        $createContentBlock->userId = Authentication::getUser()->getUserId();
+
+        // The command bus will handle the saving of the content block in the database.
+        $this->get('command_bus')->handle($createContentBlock);
+
+        return $createContentBlock;
+    }
+
+    private function getBackLink(array $parameters = []): string
+    {
+        return BackendModel::createURLForAction(
+            'ContentBlockIndex',
+            null,
+            null,
+            $parameters
+        );
+    }
+
+    private function getForm(): Form
+    {
+        $form = $this->createForm(
+            ContentBlockType::class,
+            new CreateContentBlock(),
+            ['theme' => $this->get('fork.settings')->get('Core', 'theme', 'Core')]
```

Comment:
That PR has already been made and has been merged, so you should make sure you have the latests version of 5.0.0-dev, since git saw your changes as delete and add instead of move that change would've been lost